src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -54,14 +55,24 @@ public static bool ResolveForAssembly(string assemblyLocation, ref string docume
                 return false;
             }
 
-            var qualifiedPath = Path.GetFullPath(assemblyLocation);
-            var fn = Path.GetFileNameWithoutExtension(qualifiedPath);
-            var dir = Path.GetDirectoryName(qualifiedPath);
+            var assemblyPath = Path.GetFullPath(assemblyLocation);
 
-            fn = fn + ".xml";
+            var baseDir = Path.GetDirectoryName(assemblyPath);
+            var xmlFileName = Path.GetFileNameWithoutExtension(assemblyPath) + ".xml";
 
-            documentationPath = Path.Combine(dir, fn);
+            var language = System.Threading.Thread.CurrentThread.CurrentUICulture.ToString();
+            var localizedResPath = Path.Combine(baseDir, language);
+            documentationPath = Path.Combine(localizedResPath, xmlFileName);
 
+            if (File.Exists(documentationPath))
+                return true;
+
+            localizedResPath = Path.Combine(baseDir, "en-US");
```

Comment:
By reading this we seem to be using this as a default? In which case let's promote it to a constant that we're using somewhere rather than embedding it in the code?